sbin/build.sh

```diff
@@ -1328,7 +1328,7 @@ getGradleJavaHome() {
 
   # Special case arm because for some unknown reason the JDK11_BOOT_DIR that arm downloads is unable to form connection
   # to services.gradle.org
-  if [ ${JDK11_BOOT_DIR+x} ] && [ -d "${JDK11_BOOT_DIR}" ] && [ "${ARCHITECTURE}" != "arm" ]; then
+  if [ ${JDK11_BOOT_DIR+x} ] && [ -d "${JDK11_BOOT_DIR}" ] && [ "${BUILD_CONFIG[OS_ARCHITECTURE]}" != "arm" ]; then
```

Comment:
This is correct, we should be using the "Build Arch" here, since the JDK is being run on the build host, hence we should use OS_ARCHITECTURE